docs/docs/learn/shards/.pages

```diff
@@ -1,6 +1,6 @@
 nav:
   - index.md
-  - introduction.md
   - what-is-shards.md
   - the-flow.md
   - working-with-data.md
+  
```

Comment:
new line should be empty (here it has a space which make it not being considered a new line)